.golangci.yaml

```diff
@@ -39,24 +16,63 @@ linters:
   - predeclared
   - revive
   - staticcheck
-  # https://github.com/golangci/golangci-lint/issues/2649
-  # - structcheck
-  - typecheck
   - unused
-  # https://github.com/golangci/golangci-lint/issues/2649
-  # - wastedassign
   - whitespace
-  disable:
-  # The following linters are run separately by arcanist at the moment.
-  # This is because we have autofix hooks for these linters.
-  - goimports
-  disable-all: false
-
-linters-settings:
-  errcheck:
-    # yamllint disable-line rule:line-length
-    ignore: io:Close,github.com/fatih/color,github.com/spf13/pflag:MarkHidden,github.com/spf13/viper:(BindEnv|BindPFlag),github.com/spf13/cobra:(Help|MarkFlagRequired|Usage),github.com/segmentio/analytics-go/v3:Enqueue,database/sql:Rollback,github.com/nats-io/nats.go:Unsubscribe
-  goimports:
-    local-prefixes: px.dev
-  nakedret:
-    max-func-lines: 0
+  settings:
+    errcheck:
+      exclude-functions:
+      - io.Close
+      - (*github.com/spf13/pflag.FlagSet).MarkHidden
+      - github.com/spf13/viper.BindEnv
+      - github.com/spf13/viper.BindPFlag
+      - github.com/spf13/viper.BindPFlags
+      - (*github.com/spf13/cobra.Command).Help
+      - (*github.com/spf13/cobra.Command).MarkFlagRequired
+      - (*github.com/spf13/cobra.Command).Usage
+      - (github.com/segmentio/analytics-go/v3.Client).Enqueue
+      - (*database/sql.Tx).Rollback
+      - (*github.com/nats-io/nats.go.Subscription).Unsubscribe
+    revive:
+      rules:
+      - name: unused-parameter
+        disabled: true
+    staticcheck:
+      checks:
+      - all
+      - "-ST1005"  # ignore the "ST1005: error strings should not be capitalized" check
+      - "-QF1008"  # ignore omit embedded fields from selector expression
+  exclusions:
+    generated: lax
+    presets:
+    - comments
+    - common-false-positives
+    - legacy
+    - std-error-handling
+    paths:
+    - third_party$
+    - builtin$
+    - examples$
+issues:
+  max-issues-per-linter: 0
+  max-same-issues: 0
+formatters:
+  enable:
+  - gci
+  - gofmt
+  - gofumpt
```

Comment:
gofumpt is a superset of gofmt so you can drop gofmt.

Reply:
Updated in [d0a0ce3](https://github.com/pixie-io/pixie/pull/2169/commits/d0a0ce3823abde05abb2cdaaadd7d9d33cc842d2)